docs/community/release_guide.md

```diff
@@ -117,8 +119,8 @@ pub   rsa4096/37356FA0 2022-10-27 [SC] [expires: 2028-10-27]
 uid         [ultimate] Ethan Feng <ethanfeng@apache.org>
 sub   rsa4096/D2ED4CBF 2022-10-27 [E] [expires: 2028-10-27]
 ```
-
-> Note: To follow the [Apache's release specification](https://infra.apache.org/release-signing.html#note), all new RSA keys generated should be at least 4096 bits. Do not generate new DSA keys.
+!!! note
+  To follow the [Apache's release specification](https://infra.apache.org/release-signing.html#note), all new RSA keys generated should be at least 4096 bits. Do not generate new DSA keys.
```

Comment:
I remember a blank line is required before the content, you can check the preview site locally